### ISSUE TITLE
Customer Misuse case

### ISSUE THREAD

**Issue by scummings-tech** [Hinrichsta/FA23-Cyber8420#28]
Created customer misuse case

**Comment by scummings-tech** [comment]
completed customer misuse case in place of sysAdmin misuse case

**Comment by lreichlinger** [comment]
Hey @scummings-tech,

I was not able to access your link to the file in the word document, could you please send the file over directly through Canvas or through Github so we can incorporate it into the Security Document properly. This would be the .drawio file that Henri and I helped you save the other day. It looks good in the word document as screenshots, but is not something we can pull into the security document.

**Comment by lreichlinger** [comment]
Added all as assignees incase I miss a response.

**Comment by scummings-tech** [comment]
Lucas,

I was not able to download the file in gitHub. I had to create a word document instead. The misuse case you and Henri were helping me with was for the sysAdmin and not the customer bank misuse case.  My firewall and anti-virus software is blocking the aptio website because of a vulnerability being encountered.
I responded to both you and Henri a few minutes ago in canvas. I created the misuse case in a word document. I enlisted Henri's help in recreating what I drew, he will just need to download the drawing in gitHub. I also provided an explanation of the customer scenario based upon the usecase
